base/s6-rc-modifications/pulpcore-api/run

```diff
@@ -13,4 +13,4 @@ export PULP_SETTINGS /etc/pulp/settings.py
 export HOME /var/lib/pulp/
 
 # TODO: Modify OCI base image to allow for auto reload to be configurable so we don't have to override this.
-/usr/local/bin/gunicorn pulpcore.app.wsgi:application --bind "127.0.0.1:24817" --name pulp-api --access-logfile - --access-logformat "pulp [%({correlation-id}o)s]: %(h)s %(l)s %(u)s %(t)s \"%(r)s\" %(s)s %(b)s \"%(f)s\" \"%(a)s\"" --reload
+/usr/local/bin/gunicorn pulpcore.app.wsgi:application --bind "127.0.0.1:24817" --timeout 600 --name pulp-api --access-logfile - --access-logformat "pulp [%({correlation-id}o)s]: %(h)s %(l)s %(u)s %(t)s \"%(r)s\" %(s)s %(b)s \"%(f)s\" \"%(a)s\"" --reload
```

Comment:
I think this file is not used (or was expected to) anymore. The file for that modification is `/etc/init/pulpcore-api` and we have an envvar for gunicorn timeout(`PULP_GUNICORN_TIMEOUT`).

Reply:
Here is the place where we copy the service to the container:
https://github.com/pulp/oci_env/blob/508344bc00e21c4318134e6aceaf1631c1104f9a/base/Dockerfile#L22

I think this is the right place to make the modification. After re-building the image, the changes are present in the container.

Reply:
I believe it works because we still didn't merged this yet #100

Reply:
+1 to use the envar. the default should stay as is ( which is 30 sec, i think)

Reply:
If we merge https://github.com/pulp/oci_env/pull/100, we should be able to set `--reload` and `--timeout` from the compose.yaml file. I can then close this PR along with the referenced issue.

Reply:
The value can be now set from `compose.env`  by overriding the `PULP_GUNICORN_TIMEOUT` environment variable. Therefore, I am closing the PR because of https://github.com/pulp/oci_env/commit/2cbc9ecac48c22bd923edc4d99dafc43a81e6fb7#diff-b093d34eec92adf6e0cc6e445edb7147ea45d25fb0c98f2f84a60f5aae6f34f7L22.